.gitignore

```diff
@@ -23,3 +23,9 @@ ReaderClientCert.sig
 bugsnag.conf 
 helpstack.conf 
 *.jks
+
+# backup files
+*~
+
+# all Android Studio cruft
+/.idea
```

Comment:
We do not want to ignore the `.idea` directory. Keeping this is essential to being able to load the project in Android Studio without importing.